### DISCUSSION TITLE
Checkout ✔️

### DISCUSSION THREAD

**Discussion by sFranchetto** [kennyphan100/SOEN341#88] [Show and tell]
Functioning cart checkout, UI complete with form validation.

Need to continue working on the DB table portion of orders/checkout to have more consistent data held in the DB that makes logical sense. I will start brainstorming fixes in Sprint 3 but for now, the functionality works for what we need to continue developping.

**Comment by kennyphan100** [comment]
Sounds good! The UI for the checkout page looks great! 👏👏👏
